### PR TITLE
ROX-13673 + ROX-13674 + ROX-13675: Create cidr block side panel layout

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR is an iterative step to creating the CIDR Block side panel. I decided to include three of the subtasks in this PR to minimize PRs. You can find all the subtasks for the CIDR Block side panel work here: https://issues.redhat.com/browse/ROX-13672

## Checklist
- ~[ ] Investigated and inspected CI test results~
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Screenshots

Screenshot 2022-11-30 at 10 42 59 AM

### CODE REVIEW

**Comment by sachaudh** [comment]
@alanonthegit @zhenpesky where can I find this icon?

Screenshot 2022-11-30 at 10 49 39 AM

File: ui/apps/platform/src/Containers/NetworkGraph/cidr/CidrBlockSideBar.tsx
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/cidr/CidrBlockSideBar.tsx:87]: Are you going to need all the `nodes` at a later iteration of this component? (similar question about `edges` )

If not, it may be worth having the parent compute this and past it in.

File: ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx:13]: Question that I forgot to ask earlier, about the colors of these "badges".

Is our UX team giving these colors, and will we try to get them added to PatternFly, so that all Red Hat apps can standardize on colors for k8s entities?

**Review by vjwilson — APPROVED** [review]
Approving to unblock, but would like to hear back on the few questions I had.

File: ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx:154]: Yea you have a good point. I made the change because internally we kept it as the backend values and only used the UI values in the rendered view so I made the change to allow myself to see things for now. I will change it to `CIDR_BLOCK` in a future PR. We will eventually type out the `data` field so there is a more consistent data structure

File: ui/apps/platform/src/Containers/NetworkGraph/cidr/CidrBlockSideBar.tsx
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/cidr/CidrBlockSideBar.tsx:87]: Good point. I think for now we'll keep it and decide if we need to refactor things in a future PR

File: ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx:13]: I wasn't given color values, but I used what I saw from the mocks. I'm not entirely sure about standardizing these for other apps though. For the CIDR block icon, I just tried to make it similar to what we had already. I asked a question about where to get that IP icon. I'll wait on a response from the UX team about that. @alanonthegit @zhenpesky maybe you have thoughts on this

Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx:17]: Will merge but make changes as necessary based on UX's reply

**Pull request merged**
